src/components/heading/heading.scss

```diff
@@ -31,7 +31,7 @@
 .carbon-heading__title {
   display: inline-block;
   font-weight: 900;
-  line-height: 32px;
+  line-height: 22px;
```

Comment:
Looks like reducing this has caused the text to get too close to the separator. Maybe it needs a bottom margin applied? 

This is what the spacing used to be like:

![screen shot 2017-11-10 at 10 07 13](https://user-images.githubusercontent.com/1668080/32653216-f33d9ca2-c5fe-11e7-9b02-f4b9cad0ecea.png)